### PR TITLE
Phase I

### PR DESCRIPTION
**Pull request by profxj**
Gets the code up and running

Have successfully run `test.py` which is now in `tests`

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+# Library of functions used for 
+# https://ui.adsabs.harvard.edu/abs/2021arXiv210108005J/abstract
+# and
+# https://ui.adsabs.harvard.edu/abs/2021arXiv210107998J/abstract
+# Please cite the former (the 'methods' paper) when using.
+
+
+
+Questions? Email:
+clancy.w.james@gmail.com
```
Comment (cwjames1983) [inline README.txt:11]: Maybe add your email here too X?

File: steer_cube.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
+#!/bin/bash -l
+
+
```
Comment (cwjames1983) [inline steer_cube.sh:3]: TODO: add note "# this job steering file was developed for Pawsey's Magnus computer system. See https://pawsey.org.au/systems/magnus/"

File: zdm/data/Surveys/parkes_mb_class_I.dat [hunk 3]
```diff
@@ -0,0 +1,30 @@
+FBAR  1382 #MHz
```
Comment (cwjames1983) [inline zdm/data/Surveys/parkes_mb_class_I.dat:1]: This is a file that needs to change - I used an old version with an FRB in the wrong place and the incorrect Tobs. Will be updated in phase III.

File: zdm/data/Surveys/parkes_mb_class_II.dat [hunk 4]
```diff
@@ -0,0 +1,26 @@
+FBAR  1382 #MHz
```
Comment (cwjames1983) [inline zdm/data/Surveys/parkes_mb_class_II.dat:1]: The missing FRB from parkes_mb_class_I was accidentally put in here, needs to be deleted

Comment (cwjames1983) [reply zdm/data/Surveys/parkes_mb_class_II.dat:1]: Will be updated in phase III

**Review by cwjames1983 — APPROVED** [review]
No significant changes - I just noted where files need updates I will change in my phase III.

**Comment by profxj** [comment]
merging

**Pull request merged**